mods/doors/init.lua

```diff
@@ -130,7 +130,7 @@ local transform = {
 	},
 }
 
-function _doors.door_toggle(pos, node, clicker)
+function doors.door_toggle(pos, node, clicker)
```

Comment:
Now public. Needs documentation in our `game_api.txt` file. The current changes look good.

Reply:
Agreed, this is required.

Reply:
il get this done withihn the next couple of days. cheers guys